app/controllers/validation/custom_label_update_parameters.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# rubocop:disable Naming/VariableName
+
+module Validation
+  class CustomLabelUpdateParameters
+    include ActiveModel::Validations
+
+    attr_accessor :label_id,
+                  :id,
+                  :displayLabel,
+                  :displayOnFullTextFinder,
+                  :displayOnPublicationFinder
+
+    validates :displayLabel, :id, presence: true
+    validate :ids_match?
+    validate :id_range?
+    validates :displayOnFullTextFinder,
+              inclusion: { in: [true, false], message: 'Invalid value' }
+    validates :displayOnPublicationFinder,
+              inclusion: { in: [true, false], message: 'Invalid value' }
+
+    def ids_match?
+      # label_id is what is passed in the url
+      # id is what is passed in the payload
+      # this should be done in Rails 5 using the following syntax
+      # validates :id, numericality: { only_integer:true }
+      # But unfortunately, it does not work, hence this is needed
```

Comment:
>But unfortunately, it does not work

Reply:
The way I figured that it is not working as I intended is by testing it manually in my code. ```validates :id, numericality: { only_integer:true }``` was what I tried and even if I pass a string as id, for example "abc",  it was not throwing an error. I did google about it and read somewhere that it is a regression in Rails 5, but am not able to find that link now :(